### PR TITLE
SCRD-6374 Remove translation for logging message

### PR DESCRIPTION
**Pull request by ghost**
The commit includes:
1. Removed translation of the logging messages in replace server
   process
2. Updated logMessage to append new line '\n' or no new line
   based on passed in  params (SCRD-6375)
3. Removed unused messages.
4. Allowed error message from action run to show up in the page banner

### CODE REVIEW

**Comment by guglo** [comment]
Ethan has a good idea to allow translation only to English. Will try his idea instead.

**Review by jeremy-moffitt — APPROVED** [review]
LGTM

File: src/components/PlaybookProgress.js [hunk 1]
```diff
@@ -781,11 +781,20 @@ class PlaybookProgress extends Component {
         this.updateGlobalPlaybookStatus(playbookName, playId, STATUS.FAILED);
       }
       // if failed update caller page immediately
-      this.props.updatePageStatus(STATUS.FAILED);
+      this.props.updatePageStatus(STATUS.FAILED, error);
     }
   }
 
-  logMessage = (message) => {
+  logMessage = (message, notAppendNewLine) => {
```
Comment (GarySmith) [inline src/components/PlaybookProgress.js]: `notAppendNewLine` is confusing. Perhaps we can change the signature to the positive case and default it to `true`, like:
```
logMessage = (message, addNewlineIfMissing=True) 
```

File: src/components/PlaybookProgress.js [hunk 2]
```diff
@@ -781,11 +781,20 @@ class PlaybookProgress extends Component {
...
+    // if have not specified not to append new line break
+    // then if the message does not contain a new line
+    // break append new line break
+    if (!notAppendNewLine) {
+      var hasNewLine = /\r|\n/.exec(message);
+      if (!hasNewLine) {
+        message = '\n' + message + '\n';
```
Comment (GarySmith) [inline src/components/PlaybookProgress.js]: I don't think we should have a newline before the message, or else all of our messages will be double-spaced.

**Review by GarySmith — CHANGES_REQUESTED** [review]

File: src/components/PlaybookProgress.js (hunk 2 above)
Comment (guglo) [reply src/components/PlaybookProgress.js]: @GarySmith ok will remove it

File: src/components/PlaybookProgress.js (hunk 1 above)
Comment (guglo) [reply src/components/PlaybookProgress.js]: @GarySmith  that is better. will update

**Review by GarySmith — APPROVED** [review]

**Pull request merged**